pages/neighbourhoods.py

```diff
@@ -0,0 +1,434 @@
+            {'label': 'Wassenaar', 'value': 'Wassenaar'},
+            # {'label': 'Roaz', 'value': 'Roaz'},
+            # {'label': "Haaglanden", 'value': 'Haaglanden'},
+            # {'label': 'Leiden', 'value': 'Leiden'},
+            # {'label': 'Delft', 'value': 'Delft'}
+            ],
+        value="'s-Gravenhage", 
+        className = "custom_select"
+    )
+
+layout = html.Div([
+            html.Div(
+                dbc.Accordion([
+                    dbc.AccordionItem([
+                        html.Div([
+                            html.Div([html.Label('Choose a variable to plot :', id= 'choose_variable', htmlFor= 'drop_var_id'), drop_var], id= 'select_variable'),
+                            html.Div([html.Label('Choose a region to plot:', id='choose_area', htmlFor= 'drop_municipality'), drop_municipality], id = 'select_region'),
+                            html.Div([html.Label('Choose neighbourhoods to plot:', id= 'choose_wijk', htmlFor= 'drop_municipality_spec_id'),
+                                dcc.Dropdown(
+                                    columns,
+                                    id = 'drop_municipality_spec_id',
+                                    clearable=True,
+                                    searchable=True, 
+                                    multi=True,
+                                    className= "custom_select"
+                                )
+                            ], id='select_neighbourhoods'),
+                            html.Div([
+                                "Select year of interest",
+                                dcc.Slider(step=1, id = 'slider_select_year'),
+                                dcc.Dropdown( id = 'drop_select_year', className= "custom_select") #when resolution is small, slider is no longer practical
+                            ],  id= 'sliderContainer')
+                        ], id= 'select_container'),
+                    ], title="Variable, Region and Year Selection :")
+                ], className = 'box'), id = "dashnav"
+            ),
+            html.Div([
+                html.Div([
+                    html.Div([
+                        html.H1(id='title_map'),
+                        html.P('Click on a tile to see the trendline!'), 
+                        dcc.Graph(id='map')
+                    ], className='box'),
+                    html.Div([
+                        html.H1(id='wijk_trend_label'),
+                        html.P('Click the button and legends to know more!'),
+                        
+                        dcc.Graph(id='wijk_trend_fig'),
+                    ], className='box')                        
+                ], id= 'leftcell'),    
+                html.Div([
+                    html.Div([   
+                        html.H1(id='title_bar'),           
+                        dcc.Graph(id='bar_fig'), 
+                    ], className='box')                    
+                ], id= 'rightcell')
```

Comment:
I think the size of the bar fig is not correct yet. bar fig should be around 1015 px in my monitor. 
However, I also realized that if we have different monitor resolutions, the box will be offset-ed, especially for the height of the bar fig "box". What do you think we should do about this?